requirements.txt

```diff
@@ -82,4 +82,4 @@ wcwidth==0.1.9
 Werkzeug==1.0.0
 whitenoise==5.0.1
 zipp==3.1.0
-
+django-debug-toolbar==2.2
```

Comment:
Would be good to move this into requirements/dev.in and compile it with `pip-compile requirements/dev.in`